src/shipchain_common/test_utils/httpretty_asserter.py

```diff
@@ -73,9 +73,21 @@ def _assert_call_in_list(cls, call, assertion):
             assert assertion['query'] == call['query'], \
                 f'Error: query mismatch, desired `{assertion["query"]}` returned `{call["query"]}`.'
 
+    @classmethod
+    def reset_calls(cls):
+        # If calling cls.reset(), latest_requests are unable to build.
+        # However, if calls were mocked in previous tests and were not asserted, then it can cause issues when
+        # asserting calls in a specific test.
+
+        for index, call in enumerate(cls.latest_requests):
+            if call is None:
```

Comment:
Why does this `continue` need to be called? If `call` is none, then setting it to none again should have no effect.

Reply:
It does not this was a little copy-pasta error.